### ISSUE TITLE
Organise an internal project announcement webinar

### ISSUE THREAD

**Issue by russelltrow** [Green-Software-Foundation/green-software-maturity-matrix#17]
# Context

Organise an internal webinar for GSF Member Orgs where @aecurrie can speed-walk us through how this project got started, what we hope to achieve during incubation, and how we will measure the impact/success.

https://github.com/Green-Software-Foundation/green-software-maturity-matrix/wiki/2024%E2%80%9001%E2%80%90XX:-GSF-Member-webinar

# Acceptance Criteria

- [ ] Schedule for after https://github.com/Green-Software-Foundation/green-software-maturity-matrix/issues/5
- [ ] Only open to GSF Member organisations

# Actions

- [ ] @aecurrie @NAMRATA-WOKE agree basic details (when & where)
- [ ]

**Comment by jawache** [comment]
Hia, I think an introductory webinar (Q&A?) is a great idea to get people to understand a project and decide if they want to get involved! 

What's the thinking around how we communicate this to members? An explicit dedicated email that goes out to members inviting them to this session, or a mention in one of our existing regular communications which goes out to all members with a registration link?

**Comment by aecurrie** [comment]
Yes I think a webinar is a great idea. I'm currently swamped trying to earn
some money so it might have to be Feb before we record it (given by then
I'll have to have done an educational session on it for my new course
https://maven.com/strategically-green/softwarep-iwolil)

We were thinking explicit email?

**Comment by jawache** [comment]
Ahh yes I saw you're course announcement, very exciting!!!

Let's discuss how to announce this, if we do an explicit email then we'd have to offer every project an explicit email and that's a lot of explicit emails! We normally just highlight things in our regular communication and we only send explicit emails for our main events like decarb and the hackathon to reduce noise.

But perhaps let's think differently about this, what would be a successful outcome for you for this session? More contributors? Specific buy in from member organizations to adopt? What does success look like?

This is an async project I believe so you can have external contributors right? If so then are you looking for more external eyeballs/collaborators on this?

What about a very short 2 min video which we have our producer edit down and perhaps add some graphics, and then somehow share it to our members in a way that they can comment async? Just thinking out of the box for alternatives, esp. ones that down require a meeting since that alienates large parts of our membership.

wdyt?

**Comment by aecurrie** [comment]
Sorry for the slow response. The OR book is finally submitted so I've
returned to the world ;-) That contains a link to the project and the GSF
graphic so good for bringing in externals (my suspicion is that won't be
too tricky)

Very happy to do the announcement in the general comms. Yes a 2 min video
is fine. Perhaps that's the best thing although I'd also be happy to do a
kickoff in different timezones.

I think this is something where we need as many people at least aware as
possible both inside and outside the GSF. I think Namrata has been
mentioning it and it sounds like there are a few folk already interested.
Perhaps a chat about it on EnvVars would be a good option? I haven't been
on in a while!

**Comment by jawache** [comment]
So is the goal to raise the profile of the project more generally? A winning hearts and minds tour? Hmm how about a proper open source speakers deck with nice graphics, get lots of people speaking about it at conferences?

Things like that?

Or like specifically getting companies onboard to use it so you've got hard case stufies. cause thats a different approach I think, more 1-1tlc.

**Comment by russelltrow** [comment]
Reached out to Namrata on 17th May to start scheduling

**Comment by NAMRATA-WOKE** [comment]
@russelltrow @aecurrie am I clear to understand this is an *internal webinar* for GSF members? If so, I'd recommend creating a Doodle with 4-5 day/time options to share with GSF-all to give them a 'vote' on when to host the webinar. The days/times can come from Anne and the project co-lead based on their availability. 

I can then inform everyone and specifically reach out to those who have shown an interest in the project's outcomes.

If we can target end June/early July, that would be great.

**Comment by pinireznik** [comment]
Actually, the idea was to organise an open webinar for a wider community to start promoting adoption of the GSMM. 
We could explain the main idea and discuss initial steps in the green software journey

**Comment by jawache** [comment]
@NAMRATA-WOKE that works.

@pinireznik / @aecurrie if the project is in a finished state then before public release it needs to go through a consensus approval by the WG and then ratified by the SC. I'd give plenty of time for people to review, information is passing slowly through orgs right now, maybe the internal meeting can be an exercise to explain the work to the members so they are informed before being asked to give consensus to it's publishing?

**Comment by pinireznik** [comment]
We had something else in mind - start explaining the Matrix to potential
users with a message that it is still WIP. We would mostly talk about the
first steps a typical company would want to take before trying to achieve
everything at once.

But an internal webinar for members sounds good too. Anne, what do
you think?

BTW, Asim, a bit off topic, we're using PACT Pathfinder
<https://www.wbcsd.org/Programs/Climate-and-Energy/Climate/SOS-1.5/News/Pathfinder-Framework-Version-2.0>
as Scope 3 emissions sharing standard and earlier this week I was talking
with the project PO and she didn't know anything about GSF.
Do you know about this project? Is some sort of collaboration between the
foundations relevant?
If you like, I can make an introduction and you can explore it with them
directly.

Pini

**Comment by aecurrie** [comment]
I do like discussing it as a WIP as I suspect it always will be. Things are
moving very very fast in green software right now. Of course it is already
available in a published form in the OR book

**Comment by NAMRATA-WOKE** [comment]
**@aecurrie @pinireznik Can you help me understand the desired outcome?**

Suppose the objective is to gain support to develop/refine the matrix. In that case, we should focus the event on our members because they are the group of people who can actually get into it and develop the project with you. Prini's suggestion and sharing a couple of specs for how members can contribute can be helpful.

If we aim to raise awareness about the project and, in this early stage, have people use it and offer feedback or give some insight into its effectiveness. Again, I would focus on increasing adoption within our member network as its not a graduated project, and create a use case template or survey that we can ask users to complete to capture information systematically. It might even be worth creating a Pilot for member organizations to adopt -- where we lay out the conditions for use to validate/debunk a critical assumption? 

@russelltrow curious to hear your thoughts on the above!

**Comment by jawache** [comment]
@pinireznik that actually sounds pretty interesting and valid for some of out other projects (like IF) feel free to connect them to me if you want, asim@greensoftware.foundation, cheers!

@aecurrie 
Let's chat about where this project is, I'm a little confused :/ 

From what I can see from the repo:

- The only contributor according to GH is @russelltrow!
- There have been 2 PRs start/end of last year and one is open and not merged.
- This is currently a draft consensus project in the GSF.
- There is communication about this wanting to be discussed with external people in an _adoption_ capacity, which is something we can only do when the project has consensus of the members, ratified by the steering committee. Until then the communication **externally** has got to be in terms of getting feedback and the feedback has got to come in a way that is captured in the repo (ideally as a PR but can also be an issue template). We also need to ensure DCO is setup correctly to make sure that any non-members who provide feedback are also automatically signing that they are free to provide their feedback royalty free etc... all members have already signed that in their membership agreement.

Is this project "finished", in terms of that the core team is happy with the content and ready to get to the next stage? If not then the fact there has been no updates in 6 months indicates this project has stalled and should be archived!

If it's finished the next stage is getting feedback from the members of the GSF in such a form that there feedback is recorded in the repo. I'd recommend an issue template, ask a few questions you specifically want answers to and also we should preliminary as them if in it's current form they would endorse, consent or object. That will give us some early signals, you really only want to go for consensus once you're pretty sure you'll get it. 

You'll need to deal with whatever feedback you get, esp. any people who've said they would object - update content etc...

Once we've done that internally then the next step might be to get some external feedback, same form, send people the link to specifically provide feedback in the repo. That's not necessary for internal consensus but it's good to have if you want to attempt to get this into an an ISO standard.

Ultimately once we've done a very explicit round of formally asking and recording feedback, if it all looks good we need to send it out for a consensus vote amongst all our members and again give them the opportunity to endorse, consent or object.

If there are no objections then we go for ratification by the SC. 

Once it's ratified, it's a proper, published GSF project and we can put all our energies into promoting it, sharing it etc... and if there is interest, perhaps trying for an ISO certification.

If you are not ready and want some super early feedback we can try to arrange an internal workshop, if you are "ready" as it looks like you are, then a feedback form and formal request to provide feedback is probably the right way. We can facilitate an email asking org leads to fill out a form, it's is a lot easier for everyone to action rather than joining a call, also having people submit issues is a a great way to demonstrate that this project has engagement/consensus. These requests tend to get fielded internally inside organizations, they huddle together, decide how they are going to respond and then respond (or don't!).

But ultimately I think you need to decide what your goal for this project is, if you **_don't_** want to go for consensus and have this published as a GSF product then we can perhaps help publish this as an opinion piece on our blog if that helps?

If you want to eventually go for ISO we need to be very careful about how we demonstrate who contributes to this project, ISO want to see lots of evidence of multiple organizations engaging, lots of contributors on the repo is best (which is why it's a problem this started out as a gdoc and that @russelltrow is the only contributor) but if there are lots of people giving feedback as issues on the repo that works also. For the SCI we demonstrated there were 20+ organizations who contributed.

If this eventually wants a chance to get into regulation, then that's where all our frustrating rules become a life saver - we can guarantee as much as possible that everyone who's contributed has released any IP so if it goes to regulation, no company can come back years later and ask for royalty fees. We can only do that if all contributions are evidenced through GitHub, we can prove who contributed every line of the text and also prove that that person either worked for a member org who released their IP in their membership agreement or signed a DCO which does the same for external contribution.

Makes sense?
